src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11254,13 +11287,16 @@ private ExpressionSyntax ParseExpressionContinued(ExpressionSyntax leftOperand,
                 }
             }
 
-            // From the language spec:
+            // https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md?rgh-link-date=2024-10-02T19%3A11%3A19Z#1115-conditional-operator:
```

Comment:
Does the `rgh-link-date=...` in this url do something useful? If not, consider removing it.

Reply:
ick.  will do.